design/adr/core/0001-Developer-Scripts-TaggingAndCleanup.md

```diff
@@ -0,0 +1,105 @@
+# Developers Script - Clearing and Tagging
+
+<!--ts-->
+
+- [Status](#status)
+- [Context](#context)
+- [Proposed Design](#proposed-design)
+- [Decision](#decision)
+- [Consequences](#consequences)
+- [References](#references)
+
+<!--te-->
+
+## Status
+
+Proposed
+
+## Context  
+
+`Developer Script` repository contains helper scripts used by the EdgeX developers in their everyday work, docker-compose files for all the releases, and scripts for gathering project metrics. <br />
+This repository have only one branch - `master`. 
+While docker-compose files reside in different directories named appropriately for every specific release, this is not the case with the scripts in the main directory.
+It is supposed that these scripts are ready to work with the latest version of the project, that could not be helpful for developers troubleshooting older versions. 
+Actually, significant part of the scripts are currently outdated and not applicable even with the latest version. <br />
+Here it comes the need of keeping track scripts files and theirs cleaning up and update.
+
+## Proposed Design 
+
+### Force using annotated tags
+
+By using `annotated tags` git history will be marked with the time versions have been released. Exp: git tag -a v2.0 -m "Releasing version v2.0" <br />
+Developers will have the knowledge which specific commit to use for specific EdgeX version. 
+This will make the troubleshooting possible for both - the older and the current versions of the project.
+
+### Clearing Up and Update the scripts
+Some of the scripts need to be updated in order to work with the latest version, other are not relevant at all and need to be removed.
+
+##### Current available scripts 
+```
+MODULES*
+clean_shell.bat
+mongoshell.bat
+init_shell.bat
+startdb-no-auth.bat
+startdb.bat
+clean_mongo.js
+init_mongo.js
+linux_setup.sh*
+mongoAdminOperStateUpdate.js
+run-it.sh*
+reset-dockers.sh*
+prepare-environment.sh*
+create-containers.sh*
+update-packages.sh*
+```
+
+##### Proposed changes per script 
+* [MODULES](https://github.com/edgexfoundry/developer-scripts/blob/master/MODULES) - **to be Updated** <br /> 
+Still contains java bases modules. These needs to be replace with the latest go applications
+
+* [mongoshell.bat](https://github.com/edgexfoundry/developer-scripts/blob/master/mongoshell.bat), 
+    [init_shell.bat](https://github.com/edgexfoundry/developer-scripts/blob/master/init_shell.bat),
+    [startdb-no-auth.bat](https://github.com/edgexfoundry/developer-scripts/blob/master/startdb-no-auth.bat),
+    [startdb.bat](https://github.com/edgexfoundry/developer-scripts/blob/master/startdb.bat),
+    [init_mongo.js](https://github.com/edgexfoundry/developer-scripts/blob/master/init_mongo.js),
+    [linux_setup.sh](https://github.com/edgexfoundry/developer-scripts/blob/master/linux_setup.sh) - **to be Removed**. <br /> 
+ All these scripts set up mongo database. Instead, go application `edgex-go-mongo` is supposed to be used.
```

Comment:
do we have `edgex-go-mongo`?

Reply:
Thank you for your review @cloudxxx8 
Both `edgex-go-mongo` and `edgex-mongo` do not exist. I meant to say **docker-edgex-mongo**

Reply:
Consider taking a look on PR https://github.com/edgexfoundry/edgex-docs/pull/74 as well . Logically, PR 74 needs to be reviewed first. I will add it as a comment in the PRs description